core/src/main/resources/reference.conf

```diff
@@ -4,13 +4,16 @@ kanaloa {
   default-dispatcher {
     workTimeout = 1m
 
+    # how often dispather makes ajustment accordingly
```

Comment:
I'm also a little confused to what this is?  It seems like this is being set as the `sampleRate` in the `MetricsCollectorSettings`